_pages/en/strategy-alternate-model.md

```diff
@@ -0,0 +1,360 @@
+    <td>
+    IT Security<br>
+    IITB  Governance division
+    </td>
+  </tr>
+  <tr>
+    <td>Produce alternate Audit assessment process</td>
+    <td>Produce an alternate Audit Process that favours DevOps automation capabilities, product evolution, and dedicated product teams</td>
+    <td>
+    Audit<br>
+    IITB Governance division
+    </td>
+  </tr>
+  <tr>
+    <td>Produce alternate Accessibility assessment process</td>
+    <td>Produce an alternate Accessibility Process that favours DevOps automation capabilities, product evolution, and dedicated product teams</td>
+    <td>
+    Accessibility<br>
+    IITB Governance division
+    </td>
+  </tr>
+  <tr>
+    <td>Finance</td>
+    <td>Investigate alternative funding models</td>
+    <td>
+    Engage with Finance Advisors to investigate alternative funding model while remaining compliant with the PPPM and Investment Management Policy. Such as what the Cloud CoE did with their showback model.<br><br>
+    Technical debt is a stakeholder because they are working on an evergreen funding model (e.g. tax)
+    </td>
+    <td>
+    CFOB<br>
+    EPMO<br>
+    Technical Debt
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="4">Architecture</td>
+    <td>Get endorsement from EA</td>
+    <td>Engage with Enterprise Architecture, specifically Business Architecture, Solutions Architecture, and Technical Architecture teams to seek endorsement on the Alternate Delivery Model and their contribution</td>
+    <td>
+    EA<br>
+    ARC<br>
+    Interoperability<br>
+    IITB Senior Advisors<br>
+    </td>
+  </tr>
+  <tr>
+    <td>Review technical brick process</td>
+    <td>Review technical bricks management process to allow IT Products team to add new standards (select their own technical stacks)</td>
+    <td>Technical Architecture</td>
+  </tr>
+  <tr>
+    <td>Define API approval process</td>
+    <td>Formally define API approval process to allow IT Product teams to expose their products functionality and data via APIs</td>
+    <td>
+    Interoperability<br>
+    EA
+    </td>
+  </tr>
+  <tr>
+    <td>Produce Loose Coupling architecture guidance</td>
+    <td>Produce a guidance document to clarify and guide Architects in building IT solutions making use of loose coupling architecture principles that provide IT Product teams more autonomy (e.g. Micro-Services, 12-factor app)</td>
+    <td>Solution Architecture</td>
+  </tr>
+  <tr>
+    <td>Product Management</td>
+    <td>Get endorsement from the Resource Centre</td>
+    <td>Engage with the Resource Centre to provide an alternative means to support projects, by adopting a team mentality over individuals</td>
+    <td>
+    Resource Centre<br>
+    Senior Advisors
+    </td>
+  </tr>
+  <tr>
+    <td>DevOps</td>
+    <td>Get endorsement from Operations</td>
+    <td>Engage with Enterprise Operations and BOSS to get their endorsement in moving to a DevOps model, in parallel with the conventional ones</td>
```

Comment:
In reference to my general comment about this section, perhaps we could add

- Work with EO to define metics for automated processes which would be defined as approval from respective groups (a11y and security, for example)